test/test_helper.rb

```diff
@@ -23,10 +23,14 @@ class ActiveSupport::TestCase
   def set_organization organization = organizations(:cirope)
     Current.group        = organization.group
     Current.organization = organization
+    prefix               = organization&.prefix
```

Comment:
Como ya tiene el .group arriba, si es nil ya explotó.
```suggestion
    prefix               = organization.prefix
```